display/state.py

```diff
@@ -7,90 +7,81 @@
 from threading import Thread
 
 
-class State:
-    def __init__(self, screen, keyboard):
-        self.screen = screen
-        self.keyboard = keyboard
-        self.stateCode = StateCode.MENU
-        # do menu
-    
-    def playIntro(self):
-        if self.stateCode == StateCode.MENU:
-            self.stateCode = StateCode.INTRO
-            # do intro
-        else:
-            raise EnvironmentError
+class StateMachine:
 
-    def playGame(self):
-        if self.stateCode == StateCode.INTRO:
-            self.stateCode = StateCode.PLAYING
-            sprites = pygame.sprite.Group()
+    def __init__(self):
 
-            zucc = ZUCC()
```

Comment:
[SEVERE WARNING] Further removal of references to the ZUCC may not be tolerated.